pkg/security/probe/probe_ebpf.go

```diff
@@ -1097,7 +1097,8 @@ func (p *EBPFProbe) handleEvent(CPU int, data []byte) {
 				} else {
 					pid, err := utils.TryToResolveTraceePid(event.ProcessContext.Process.Pid, event.PTrace.NSPID)
 					if err != nil {
-						seclog.Errorf("PTrace err: %v", err)
+						seclog.Tracef("PTrace tracee resolution error for process %s in container %s: %v",
+							event.ProcessContext.Process.FileEvent.FilepathStr, containerID, err)
```

Comment:
```suggestion
							event.ProcessContext.Process.FileEvent.PathnameStr, containerID, err)
```

Reply:
As discussed, I've split the PR in 2, here's only the lowering of the log level